tasks/ca/hierarchy.py

```diff
@@ -0,0 +1,136 @@
+from tasks.hierarchy import (DenormalizedHierarchy, Hierarchy, HierarchyChildParentsUnion, HierarchyInfoUnion,
+                             HierarchyChildParent, GetParentsFunction, LevelHierarchy, LevelInfo)
+from tasks.ca.statcan.geo import (GEO_PR, GEO_CD, GEO_FSA, GEO_CMA, GEO_CSD, GEO_DA, GEO_CT, GEO_DB,
+                                  Geography)
+from lib.logger import get_logger
+
+LOGGER = get_logger(__name__)
+COUNTRY = 'ca'
+
+
+def _levels():
+    return [GEO_DB, GEO_CT, GEO_DA, GEO_CSD, GEO_CMA, GEO_FSA, GEO_CD, GEO_PR]
+
+
+class CADenormalizedHierarchy(DenormalizedHierarchy):
+    @property
+    def _country(self):
+        return COUNTRY
```

Comment:
I did this for my own curiosity. It turns out you could override this with a simple variable, e.g: `_country = COUNTRY`, instead of overriding the property method.

See:
```
class Parent:
    @property
    def _prop(self):
        raise NotImplementedError('_CountryTask must define country()')
    
    def run(self):
        print(self._prop)

class Child(Parent):
    _prop = 'Hello world!'

Parent().run() -> Exception
Child().run() -> Hello world!
```

Not saying to do this, but I found it very interesting.

Reply:
I wouldn't change it, as overriding methods seems more natural. In addition, in the future, I think that this should be handled with configuration instead of inheritance.